### DISCUSSION TITLE
Connect with Zapier!

### DISCUSSION THREAD

**Discussion by valmoz** [fattureincloud/api#395] [Announcements]
🇮🇹
Buongiorno a tutti,
Abbiamo finalmente rilasciato la versione Beta del nostro connettore Zapier!
Zapier è una Web Automation App, che permette di connettere tra loro diverse applicazioni creando dei veri e propri flussi di dati, il tutto senza scrivere una sola riga di codice!
Il nostro connettore permette finalmente di aggiungere Fatture in Cloud all'elenco delle applicazioni disponibili, ed è una delle funzionalità più richieste dai nostri utenti... Speriamo possa essere una sorpresa gradita!

La notizia completa è disponibile [qui](https://developers.fattureincloud.it/blog/zapier)!

Il Team API di Fatture in Cloud

🇬🇧
Hello everybody,
We finally released the Beta Version of our Zapier Connector!
Zapier is a Web Automation App, that makes it possible to connect a wide range of applications creating data flows, without writing a single line of code!
Our connector finally adds Fatture in Cloud to the available applications list, and it is one of the most requested functionalities by our users! We hope it is a welcome surprise!

The detailed news is available [here](https://developers.fattureincloud.it/blog/zapier)!

The Fatture in Cloud API Team

**Comment by Dreamerscommunity** [comment]
Ciao @valmoz, super contento di questa news e complimenti per il vostro lavoro. 

Sto cercando di connettere Stripe con Fatture in Cloud tramite Zapier, con l'obiettivo di inviare direttamente la fattura elettronica nel cassetto fiscale di un utente al momento della sottoscrizione di una membership sul nostro sito.

Chi acquista potrebbe essere sia un privato (quindi lo SDI di default dovrebbe essere 000000) oppure un'azienda (l'utente lo potrà scegliere su Stripe, al momento del pagamento, inserendo quindi il proprio SDI) 

Ci sono alcuni campi che non saprei come compilare lato Zapier, ed in particolare

- The Id of the document (required)
- Value of TipoCassa Used (optional)
- If set to true the invoice will not be sent to the SDI


Potresti aiutarmi a configurarli?

Grazie davvero e a presto

Marco

**Reply by valmoz** [reply]
Buongiorno @Dreamerscommunity 
grazie per i complimenti, fa sempre piacere vedere che i propri sforzi sono apprezzati.

Le risponderemo sicuramente, però le chiederei di aprire una nuova discussione con la stessa domanda, in modo tale da permettere un più facile recupero delle informazioni da parte degli altri utenti se necessario.

Grazie
Mauro

**Reply by Dreamerscommunity** [reply]
@valmoz Fatto, grazie mille a te!

**Reply by valmoz** [reply]
Per tracciamento, riporto il link alla nuova discussione
https://github.com/fattureincloud/api/discussions/446
